include/rocksdb/cloud/cloud_file_system.h

```diff
@@ -299,11 +299,12 @@ class CloudFileSystemOptions {
   bool resync_on_open;
 
   // Experimental option!
-  // This option only affects how resync_on_open works. If resync_on_open is true,
-  // and resync_manifest_on_open is true, besides fetching CLOUDMANFIEST from s3,
-  // we will fetch latest MANIFEST file as well.
-  // 
-  // This is a temporary option to help quickly rollback the change if something unexpected is wrong.
+  // This option only affects how resync_on_open works. If resync_on_open is
+  // true, and resync_manifest_on_open is true, besides fetching CLOUDMANFIEST
+  // from s3, we will fetch latest MANIFEST file as well.
+  //
+  // This is a temporary option to help quickly rollback the change if something
+  // unexpected is wrong.
```

Comment:
are all these changes to comments occuring because you do not have the correct tab setting? is there a way to avoid these?

Reply:
I believe this is the correct formatting. I've ran `clang-format` on the entire file. Some previous commits left it this way either because people didn't run formatter and/or because `clang-format` behaviour sometimes changes between versions.

Reply:
Seems like there is a GH check for formatting and it's unhappy about a couple of other files as well, I'll open a separate PR that fixes them all.